amqp_broker.go

```diff
@@ -71,17 +71,23 @@ func NewAMQPConnection(host string) (*amqp.Connection, *amqp.Channel) {
 }
 
 // NewAMQPCeleryBroker creates new AMQPCeleryBroker
-func NewAMQPCeleryBroker(host string) *AMQPCeleryBroker {
-	return NewAMQPCeleryBrokerByConnAndChannel(NewAMQPConnection(host))
+func NewAMQPCeleryBroker(host string, queue ...string) *AMQPCeleryBroker {
+	queueName := DefaultQueueName
+	if len(queue) > 0 && queue[0] != "" {
+		queueName = queue[0]
+	}
+
+	conn, ch := NewAMQPConnection(host)
+	return NewAMQPCeleryBrokerByConnAndChannel(conn, ch, queueName)
 }
 
 // NewAMQPCeleryBrokerByConnAndChannel creates new AMQPCeleryBroker using AMQP conn and channel
-func NewAMQPCeleryBrokerByConnAndChannel(conn *amqp.Connection, channel *amqp.Channel) *AMQPCeleryBroker {
+func NewAMQPCeleryBrokerByConnAndChannel(conn *amqp.Connection, channel *amqp.Channel, queueName string) *AMQPCeleryBroker {
```

Comment:
this also breaks backward compatibility

Reply:
This one does break compatibility, I didn't realized it when writing it.